types/index.esm.d.ts

```diff
@@ -38,14 +38,14 @@ export {
   ScriptableAndArrayOptions
 } from './scriptable';
 
-export interface ScriptableContext {
+export interface ScriptableContext<TParsedData> {
+	active: boolean;
 	chart: Chart;
-	parsed: unknown[];
-	raw: unknown[];
 	dataIndex: number;
 	dataset: ChartDataset;
 	datasetIndex: number;
-	active: boolean;
+	parsed: TParsedData;
```

Comment:
I'm not sure why `parsed` and `raw` were arrays before. https://github.com/chartjs/Chart.js/blob/master/src/core/core.datasetController.js#L165-L166 always sets them as single values.

We likely need to improve this context definition to account for the fact that `parsed` and `raw` are only present when `type == 'data'`